ocw/lib/k8s.py

```diff
@@ -16,3 +17,27 @@ def clean_jobs(provider: Provider, client: BatchV1Api, cluster_name: str):
             else:
                 provider.log_info(f"Skip deleting from {cluster_name} the job {job.metadata.name} " +
                                   f"with age {age} (days)")
+
+def clean_helm_namespaces(provider: Provider, client, cluster_name: str):
+    now = datetime.now(timezone.utc)
+    client = client.CoreV1Api()
+    try:
+        # Retrieve the list of all namespaces
+        namespaces = client.list_namespace(watch=False)
+
+        # Print the names of all namespaces
+        while namespaces.items:
+            for ns in namespaces.items:
+                provider.log_dbg(f"Namespace: {ns.metadata.name}, Created: {ns.metadata.creation_timestamp}")
+                age = (now - ns.metadata.creation_timestamp).days
+                if (ns.metadata.name.startswith('helm-test') and age > 7):
+                    try:
+                        # Delete the namespace
+                        client.delete_namespace(ns.metadata.name)
```

Comment:
you need to obey `provider.dry_run` option , please checkout how it is done in `cleanup_jobs` above and do the same

Reply:
I did so, please re-check.